examples/fredrik/dask/pipeline.yml

```diff
@@ -0,0 +1,16 @@
+version: 1
+pipeline:
+  image: backtickse/sparky-test
+  base: backtickse/task-spark
+
+  cluster:
+    default: docker
+    docker:
+      network: tasks
+    kubernetes:
+      namespace: default
+      domain: cluster.backtick.se
+  
+  environment:
+    AWS_ACCESS_KEY_ID: AKIASMZJSEC3JBSJKUMZ
+    AWS_SECRET_ACCESS_KEY: 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
```

Comment:
friendly ping, @OskarHandmark are these real secrets?

Reply:
Nah, all keys were rolled a few months ago.

Reply:
Nah, all keys were rolled a few months ago.

Reply:
it probably makes sense to remove some of this code tho

Reply:
Some one might get high expectation on free compute for their bitcoin mining operations.